foundry.toml

```diff
@@ -4,7 +4,10 @@ out = 'out'
 build_info = true
 build_info_path = 'out/build-info'
 solc_version = "0.8.19"
+evm_version = 'cancun'   # Some of the test cases failing on AaveStrategy's test cases are working fine by explicitly mentioning this.
 
 libs = ['node_modules', 'lib']
 test = 'test'
-cache_path  = 'cache_forge'
\ No newline at end of file
+cache_path  = 'cache_forge'
+[rpc_endpoints]
+arbitrum = 'https://arb1.arbitrum.io/rpc'
```

Comment:
What is the significance of this, where is this used?

Reply:
@parv3213 i was referring to [this](https://book.getfoundry.sh/cheatcodes/rpc), If that has no effect, I can remove it

Reply:
If we are not using it, we can remove it.

Reply:
Fixed